### DISCUSSION TITLE
Need advice to debug latest commits in visual studio 2022 ?

### DISCUSSION THREAD

**Discussion by Zbuffers** [AssetRipper/AssetRipper#1105] [General]
Hi,
Since the 'old' gui has been removed, it seams i cannot launch the app in either Release or Debug mode.

Prior to this commit, I could launch the electron.GUI app  just fine in VS, (there was a setting to launch a .NET exe, it i remember correctly)
But now, when I launch it, (Debug button in VS) it launches my web browser: I see the start page,
but i don't see any way of loading files, and if I go to the setting page and save. I get an exception : cannot connect to localhost....
and I don't see any option to launch an exe anymore.

Are there any settings I should set in VS ? 
Should I install a web server ?

I indeed have electronize installed and working.
I also tried to launch the bat file provided (start.bat), but then I get an error...
I must admit I am a real noob when it comes to Electron...

Any advice much appreciated ;)
Regards,
Z

**Comment by ds5678** [comment]
https://github.com/AssetRipper/AssetRipper/blob/master/.github/CONTRIBUTING.md#compiling
